pkg/utils/build/image.go

```diff
@@ -21,7 +21,7 @@ func ExtractImage(image string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("cannot parse docker pull spec (image) %s, error: %+v", image, err)
 	}
-	tmpDir, err := os.MkdirTemp(os.TempDir(), "sbom")
+	tmpDir, err := os.MkdirTemp(os.TempDir(), "eimage")
```

Comment:
Why `eimage`?

Reply:
It is the prefix of the temp directory created, earlier this function was used only while checking sbom, since we are extracting the image in that directory and is not related to `sbom` anyway, renamed it to `eimage` so that it sounds logical.

Reply:
short form of `extracted image`

Reply:
I think full form `extracted-image` would be much clearer.